pFUnit currently broken with Flang

In trying to build with Flang today I got:

```
[ 70%] Building Fortran object fArgParse/tests/CMakeFiles/fargparse_tests.dir/fargparse_tests_driver.F90.o
cd /discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests && /discover/nobackup/projects/gmao/SIteam/comp/SLES-15/llvm-flang/2025-10-14/bin/flang -D_TEST_SUITES="</discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests.inc>" -I/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/mod/fargparse_tests -I/discover/nobackup/mathomp4/GFE-Flang/GFE/fArgParse/tests -I/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/src -I/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/mod -I/discover/nobackup/mathomp4/GFE-Flang/GFE/build/gFTL/include/v2 -I/discover/nobackup/mathomp4/GFE-Flang/GFE/gFTL/include/v2 -I/discover/nobackup/mathomp4/GFE-Flang/GFE/build/gFTL-shared/src/v2/mod -I/discover/nobackup/mathomp4/GFE-Flang/GFE/install/PFUNIT-4.13/include -O0 -g -cpp -module-dirmod/fargparse_tests -c /discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests_driver.F90 -o CMakeFiles/fargparse_tests.dir/fargparse_tests_driver.F90.o
error: Could not scan /discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests_driver.F90
/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests_driver.F90:28:10: error: #include </discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests.inc>: expected name of file to include
  #include _TEST_SUITES
           ^^^^^^^^^^^^
/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests_driver.F90:36:12: error: #include </discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests.inc>: expected name of file to include
  #  include _TEST_SUITES
             ^^^^^^^^^^^^
/discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests_driver.F90:44:12: error: #include </discover/nobackup/mathomp4/GFE-Flang/GFE/build/fArgParse/tests/fargparse_tests.inc>: expected name of file to include
  #  include _TEST_SUITES
             ^^^^^^^^^^^^
make[2]: *** [fArgParse/tests/CMakeFiles/fargparse_tests.dir/build.make:129: fArgParse/tests/CMakeFiles/fargparse_tests.dir/fargparse_tests_driver.F90.o] Error 1
make[2]: Leaving directory '/gpfsm/dnb34/mathomp4/GFE-Flang/GFE/build'
make[1]: *** [CMakeFiles/Makefile2:9087: fArgParse/tests/CMakeFiles/fargparse_tests.dir/all] Error 2
make[1]: Leaving directory '/gpfsm/dnb34/mathomp4/GFE-Flang/GFE/build'
make: *** [Makefile:166: all] Error 2
```

I'm trying to debug now.

--- a/ChangeLog.md
+++ b/ChangeLog.md
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix parser on Windows for paths with different drive letters
 - Updates for CMake versions newer than 3.30.
 - Update NVHPC CI (build only)
+- Fix CMake for LLVM Flang
 
 ## [4.13.0] - 2025-09-30
 

--- a/include/add_pfunit_ctest.cmake
+++ b/include/add_pfunit_ctest.cmake
@@ -97,7 +97,13 @@ function (add_pfunit_ctest test_package_name)
     endif (${should_write_inc_file})
   endif ()
 
-  target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES=\<${test_suite_inc_file}\>)
+  # LLVM Flang apparently does not like the <...> syntax for includes?
+  if (CMAKE_Fortran_COMPILER_ID STREQUAL "LLVMFlang")
+    target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES="${test_suite_inc_file}")
+  else()
+    target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES=\<${test_suite_inc_file}\>)
+  endif()
+
   target_include_directories (${test_package_name} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 
   if (PF_TEST_EXTRA_USE)

--- a/include/add_pfunit_test.cmake
+++ b/include/add_pfunit_test.cmake
@@ -102,7 +102,11 @@ function (add_pfunit_test test_package_name test_sources extra_sources extra_sou
     target_include_directories (${test_package_name} PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/include/${test_package_name})
 
     # Define TestSuites
-    target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES=\<${TEST_SUITE_INC_FILE}\>)
+    if (CMAKE_Fortran_COMPILER_ID STREQUAL "LLVMFlang")
+      target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES="${test_suite_inc_file}")
+    else()
+      target_compile_definitions (${test_package_name} PRIVATE _TEST_SUITES=\<${test_suite_inc_file}\>)
+    endif()
 
     # Test utility preprocessing
     set_property ( SOURCE ${PFUNIT_TESTUTILS}

